app/src/main/res/layout/activity_main_page.xml

```diff
@@ -379,6 +387,7 @@
             android:contentDescription="@string/help_button"
             android:onClick="goToHelpParametersActivity" />
```

Comment:
This Button seems not to have any vertical layout.

I think we can make it bigger and give it a marginBottom.

This way it will be easier on case of emergency.